tests/unittests/clients/studio/resources/chat/test_chat_completions.py

```diff
@@ -35,10 +47,204 @@ async def test_async_chat_create__when_bad_import_to_chat_message__raise_error()
 
 
 def test__when_model_and_model_id__raise_error():
-    client = AI21Client()
+    client = AI21Client(
+        api_key=_DUMMY_API_KEY,
+    )
     with pytest.raises(ValueError):
         client.chat.completions.create(
-            model="jamba-instruct",
+            model="jamba-1.5",
             model_id="jamba-instruct",
             messages=[ChatMessage(role=RoleType.USER, text="Hello")],
         )
+
+
+# ----------------------------------- Basic Happy Flow: ----------------------------------- #
+
+basic_happy_flow_response_json = {
+    "id": "chat-cc8ce5c05f1d4ed9b722123ac4a0f267",
+    "choices": [
+        {
+            "index": 0,
+            "message": {"role": "assistant", "content": " Hello! How can I assist you today?"},
+            "finish_reason": "stop",
+        }
+    ],
+    "usage": {"prompt_tokens": 144, "completion_tokens": 14, "total_tokens": 158},
+    "meta": {"requestDurationMillis": 236},
+}
+
+basic_happy_flow_expected_content = " Hello! How can I assist you today?"
```

Comment:
We use terminology like - `_DUMMY_HAPPY_FLOW_RESPONSE_JSON` for this kind of testing contsts